frontend/packages/console-app/src/components/quick-starts/controller/QuickStartFooter.tsx

```diff
@@ -22,46 +24,75 @@ const QuickStartFooter: React.FC<QuickStartFooterProps> = ({
   onNext,
   onBack,
   footerClass,
+  quickStartId,
 }) => {
   const location = useLocation();
   const { pathname: currentPath } = location;
   const quickStartPath = '/quickstart';
   const { t } = useTranslation();
 
+  const { restartQuickStart } = React.useContext<QuickStartContextValues>(QuickStartContext);
+
   const PrimaryButtonText = {
-    START: t('quickstart~Start tour'),
+    START: t('quickstart~Start'),
+    CONTINUE: t('quickstart~Continue'),
     NEXT: t('quickstart~Next'),
     CLOSE: t('quickstart~Close'),
   };
 
-  const getPrimaryButtonText = React.useCallback((): string => {
+  const SecondaryButtonText = {
+    BACK: t('quickstart~Back'),
+    RESTART: t('quickstart~Restart'),
+  };
+
+  const onRestart = React.useCallback(
+    (e: React.SyntheticEvent) => {
+      e.preventDefault();
+      e.stopPropagation();
+      restartQuickStart(quickStartId, totalTasks);
+    },
+    [quickStartId, restartQuickStart, totalTasks],
+  );
+
+  const getPrimaryButtonText = () => {
+    if (status === QuickStartStatus.NOT_STARTED) return PrimaryButtonText.START;
+
     if (taskNumber === totalTasks) return PrimaryButtonText.CLOSE;
 
     if (taskNumber > -1 && taskNumber < totalTasks) return PrimaryButtonText.NEXT;
 
-    return PrimaryButtonText.START;
-  }, [
-    PrimaryButtonText.CLOSE,
-    PrimaryButtonText.NEXT,
-    PrimaryButtonText.START,
-    taskNumber,
-    totalTasks,
-  ]);
+    return PrimaryButtonText.CONTINUE;
+  };
 
-  return (
-    <div className={`co-quick-start-footer ${footerClass}`}>
+  const getPrimaryButton = (
+    <Button
+      style={{
+        marginRight: 'var(--pf-global--spacer--md)',
+      }}
+      type="submit"
+      variant="primary"
+      onClick={onNext}
```

Comment:
If pressed on `Continue` button, should go the next unstarted step

Reply:
updated